original/java/src/test/java/PageDBTest.java

```diff
@@ -0,0 +1,12 @@
+import org.dbpedia.categories.database.PageDB;
+
+public class PageDBTest {
+
+	public static void main(String[] argv){
```

Comment:
can you add some explanation on what this code does? it is on the test folder but not very clear.